File: app/src/main/java/com/example/guesstheword/AlphabetFragment.java
package com.example.guesstheword;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

public class AlphabetFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_alphabet, container, false);
        String buttonId = getArguments().getString("buttonId");


        return rootView;
    }
}

File: app/src/main/java/com/example/guesstheword/AlphabetDir/AlphabetFragment.kt
package com.example.guesstheword

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

class AlphabetFragment : Fragment() {
    private var _binding: AlphabetFragment ?= null


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.fragment_alphabet, container, false)

        val letter = arguments?.getString("letter")


        return view
    }
}